### ISSUE TITLE
Parameters for fine-tune

### ISSUE THREAD

**Issue by HoracceFeng** [pjreddie/darknet#840]
Hi all,

Recently I am trying to figure out a model for detection with 100 classes. My image size is 512*512 and ground truth box vary from 10 pixel to around 100 pixel. Some parameters have changed for fine-tuning as below:

## [net]
# Testing
batch=24 
subdivisions=2
# Training
# batch=64
# subdivisions=16
width=512 
height=512
channels=3
momentum=0.9
decay=0.0005
angle=0.2     #0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.000008
burn_in=1000
max_batches = 502000
policy=steps
steps=25000,50000,80000      #400000,450000
scales=.5,.5,.1              #.1,.1

The classification Module:
[convolutional]
size=1
stride=1
pad=1
filters=678
activation=linear

[yolo]
mask = 0,1,2
anchors = 16,18, 22 25, 30,36, 41,47, 55,58, 68,79, 92,95, 117,132, 183,178
classes=221
num=9
jitter=.3
ignore_thresh = .5
truth_thresh = 1
random=1

Now the problem is, I just got a so-so model. Many bounding box are not tight enough to the target and the confidence is relatively slow, I have trained more than 80000 steps and now the loss is about 0.02, pretty low and stable right now. I wonder if I can change some parameters to at least improve the confidence such as change the ignore_thresh from .5 to .75. Another problem is, the detector cannot perform well when facing with dense objects sample, don't know how to solve it. It seems like the confidence is pretty slow if several objects are aligned and close (they are not overlap, and I have tried changing NMS when testing, not work) 

BTW, in #531 , @ahsan856jalal  said changing the width of detector may do good to small object detection, I am now trying this but still don't know if it works.

Any idea?
